### DISCUSSION TITLE
Attribute shorthand class ignores all but the first class

### DISCUSSION THREAD

**Discussion by seunggs** [markdoc/markdoc#335] [Q&A] [answered]
### What happened?

In a custom tag, trying to have more than one class does not work - only the first class takes effect and the rest of the classes are ignored.

### To reproduce

Create a simple custom tag that wraps the children in `div` and try to add more than one class.

```
{% div .mb-3 .text-teal-500 %}test{% /div %}
```

Class `mb-3` works fine, but `text-teal-500` is ignored.

### Version

^0.2.2

**Comment by seunggs** [comment, marked as answer]
Hmm - this actually seems somehow related to tailwindcss. For example, `text-sky-500` works. 

For some reason, some of the classes are ignored inside Markdoc. I noticed a similar behavior with `grid` classes.

**Reply by mfix-stripe** [reply]
Hey @seunggs, I imagine this is just an issue with your custom `div` tag implementation. 

You can see from this example that multiple classes work in Markdoc: https://markdoc.dev/sandbox?mode=preview&c=IyBIZWFkaW5nIHslIC5jbGFzczEgLmNsYXNzMiAlfQ%253D%253D
